bin/install-kernel-headers.sh

```diff
@@ -3,20 +3,24 @@
 # Get the kernel headers for this linux version
 # Yoinked from https://github.com/iovisor/bpftrace/blob/master/INSTALL.md#kernel-headers-install
 
-set -e
+set -ex
 
 KERNEL_VERSION="${KERNEL_VERSION:-$(uname -r)}"
 kernel_version="$(echo "${KERNEL_VERSION}" | awk -vFS=- '{ print $1 }')"
 major_version="$(echo "${KERNEL_VERSION}" | awk -vFS=. '{ print $1 }')"
 
-apt-get install -y build-essential bc curl flex bison libelf-dev
+apt-get install -y build-essential bc curl flex bison libelf-dev libssl-dev
```

Comment:
Not sure if this is a Fedora-specific thing or not, but I had to install the `libssl-dev` package too